flake.nix

```diff
@@ -0,0 +1,11 @@
+{
+  description = "Development environment for Space Station 14";
+
+  inputs.nixpkgs.url = "github:NixOS/nixpkgs/release-23.11";
+  inputs.flake-utils.url = "github:numtide/flake-utils";
+
+  outputs = { self, nixpkgs, flake-utils }:
+    flake-utils.lib.eachDefaultSystem (system:
```

Comment:
Instead of pulling in flake-utils (which personally I find overkill given the use here) you could do something like in here: https://github.com/space-wizards/SS14.Launcher/blob/master/flake.nix

Reply:
Thanks for the tip!